test/ng/parseSpec.js

```diff
@@ -2793,7 +2793,7 @@ describe('parser', function() {
           var fn = $parse('a');
           expect(fn.assign).toBeTruthy();
           var scope = {};
-          fn.assign(scope, 123);
+          expect(fn.assign(scope, 123)).toBe(123);
```

Comment:
If we are super nitpicky, this is out of scope for this test (i.e. it should be its own it). But it's no dealbreaker